src/Normalizer/AssessorRequestExportNormalizer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace AppBundle\Normalizer;
+
+use AppBundle\Entity\AssessorRequest;
+use AppBundle\Serializer\XlsxEncoder;
+use AppBundle\Utils\PhoneNumberFormatter;
+use Symfony\Component\Intl\Intl;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class AssessorRequestExportNormalizer implements NormalizerInterface
+{
+    /** @var TranslatorInterface */
```

Comment:
docblock not needed since the typehint of this property is made by the constructor